src/ooxml/testcases/org/apache/poi/xssf/extractor/TestXSSFImportFromXML.java

```diff
@@ -85,7 +87,7 @@ public void testImportFromXML() throws IOException, XPathExpressionException, SA
 		}
 	}
 	
-	@Test(timeout=60000)
+	@Test(timeout=60000_00)
```

Comment:
Why multiply by 100?

Reply:
Sorry, that was a hack to avoid that the test is terminated to early while debugging. I forgot to change it back. I know that JUnit Timeout rules can be deactivated while in debug mode, but I didn't find any way to do the same for timeouts defined in the annotation.